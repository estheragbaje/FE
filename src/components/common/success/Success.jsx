import React from 'react';
import { Container } from './styles';
import image from '../../../assets/images/success.svg';

function Success() {
  return (
    <Container>
      <div>
        <p>
          Thank you for taking your time to respond to the survey, the effort is
          greatly appreciated
        </p>
      </div>
      <div>
        <img src={image} alt="success" load="lazy" />
      </div>
    </Container>
  );
}

export default Success;
