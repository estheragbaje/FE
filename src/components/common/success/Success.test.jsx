import React from 'react';
import { shallow } from 'enzyme';

import Success from './Success';

describe('<Success />', () => {
  it('renders the Success component correctly', () => {
    const wrapper = shallow(<Success />);
    expect(wrapper).toMatchSnapshot();
  });
});
