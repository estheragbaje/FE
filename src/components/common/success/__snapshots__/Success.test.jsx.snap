// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Success /> renders the Success component correctly 1`] = `
<styled.div>
  <div>
    <p>
      Thank you for taking your time to respond to the survey, the effort is greatly appreciated
    </p>
  </div>
  <div>
    <img
      alt="success"
      load="lazy"
      src="success.svg"
    />
  </div>
</styled.div>
`;
